app/src/data/apiDefinition.ts

```diff
@@ -33,27 +33,20 @@ export function getApi({
     origin = '',
     fetchOptions,
 } : GetApiParams) {
-    const processRequestLocal = <ResponseData = unknown>(
-        // Rest operator is used to avoid duplicating parameters of `IProcessRequest` function.
-        ...requestParams: Parameters<IProcessRequest<ResponseData>>
-    ) => {
-        const [
-            url,
-            method,
-            data,
-            options,
-        ] = requestParams;
-
+    const processRequestLocal: IProcessRequest = (url, method, data, options) => {
```

Comment:
Do we need `processRequestLocal` here? Probably we can use native processRequest method